client/src/components/Cart/Item.js

```diff
@@ -1,17 +1,50 @@
 import "../Cart/Item.css";
+import { useNavigate } from "react-router-dom";
+import { useDispatch } from "react-redux";
+import QuantityButton from "../ProductPage/QuantityButton";
+import { removeProductFromCart } from "../../redux/cart/cartSlice";
+import { useEffect } from "react";
 
 function Item({ item }) {
+  const navigate = useNavigate();
+  const dispatch = useDispatch();
+
   return (
     <div id="item-container">
-      <div className="image-container">
-        <img className="item-image" src={item.image} alt="Broken link" />
+      <div
+        className="image-container"
+        onClick={() => {
+          navigate("/product");
+        }}
+      >
+        <img
+          className="item-image"
+          src={item.productDetails.imgURLs[0]}
+          alt="Broken link"
```

Comment:
suggestion: maybe add a photo for all the broken link?

Reply:
can only add text for the alt attribute :(